src/components/ProjectSettingsPage/index.jsx

```diff
@@ -97,67 +103,50 @@ class ProjectSettingsPage extends React.Component {
     const { projectName, projectDescription } = this.state;
     const {
       updateProject,
-      name,
-      description,
       match: {
         params: { projectID },
       },
     } = this.props;
+    const projectInfo = JSON.parse(localStorage.getItem("project"));
+    const { name, description } = projectInfo;
+
     const trimmed = (input) => input.trim();
     const trimprojectName = trimmed(projectName);
     const trimprojectDescription = trimmed(projectDescription);
 
     if (trimprojectName !== name || trimprojectDescription !== description) {
       if (!trimprojectName || !trimprojectDescription) {
         this.setState({
-          error: "please provide either a new name or description",
+          error: "all fields are required",
```

Comment:
Hey @khalifan-kfan, I dont think there is any point under project update where this will be shown.
Because non of the fields should be required

Reply:
@Mubangizi this is such a scenario 
![Screenshot from 2021-08-20 09-56-01](https://user-images.githubusercontent.com/32802973/130193005-33f51237-ad1c-4133-9b41-19ef977f5ab1.png)

Reply:
@Mubangizi  the fields  are initially  populated  with existing  data and this is triggered  wen one of the fields  is clear  completely an the user is trying  to submit empty  space.
What would you suggest  this error  is changed to  or it should  be removed completely?

Reply:
We can not remove it, as it is an edge case.